template/package.json

```diff
@@ -115,9 +116,8 @@
         "@semantic-release/git",
         {
           "assets": [
-            "./CHANGELOG.md",
-            "package.json",
-            "./docs/api.md"
+            "./**/*.md",
```

Comment:
Same here... I think it's safest that we just manually manage the expected files and test later how this works with an individual repo if the globbing works or not.

Reply:
Sure, thats reasonable.